### PR TITLE
[Workers] Module Workers docs

### PR DESCRIPTION
**Pull request by deadlypants1973**
PCX-6742

- Service Worker syntax page has been deleted
- Module Worker page has been updated with bindings info and service worker info
- Module Worker page's old dashboard content on ESM has been deleted as it is no longer relevant

TODO:
- redirects + link updates

### CODE REVIEW

File: content/workers/learning/migrate-to-module-workers.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+---
+pcx_content_type: concept
+title: Migrate to module Workers
+weight: 9
+---
+
+# Migrate to module Workers
+
+This guide will show you how to migrate your Workers from the [Service Worker](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API) syntax to the new [Module Worker](https://blog.cloudflare.com/workers-javascript-modules/) format.
```
Comment (irvinebroque) [inline content/workers/learning/migrate-to-module-workers.md]:
```suggestion
This guide will show you how to migrate your Workers from the [Service Worker](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API) foramt to the [ES Modules](https://blog.cloudflare.com/workers-javascript-modules/) format. 
```

File: content/workers/learning/migrate-to-module-workers.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
+---
+pcx_content_type: concept
+title: Migrate to module Workers
+weight: 9
+---
+
+# Migrate to module Workers
```
Comment (irvinebroque) [inline content/workers/learning/migrate-to-module-workers.md]:
```suggestion
# Migrate from the Service Worker to ES Module format
```

File: content/workers/learning/migrate-to-module-workers.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
+---
+pcx_content_type: concept
+title: Migrate to module Workers
```
Comment (irvinebroque) [inline content/workers/learning/migrate-to-module-workers.md]:
```suggestion
title: Migrate from the Service Worker to ES Module format
```

File: content/workers/learning/migrate-to-module-workers.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+weight: 9
+---
+
+# Migrate to module Workers
+
+This guide will show you how to migrate your Workers from the [Service Worker](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API) syntax to the new [Module Worker](https://blog.cloudflare.com/workers-javascript-modules/) format.
+
+## Advantages of migrating
+
+There are several reasons you might want to migrate your Workers to the module syntax:
```
Comment (irvinebroque) [inline content/workers/learning/migrate-to-module-workers.md]:
```suggestion
There are several reasons you might want to migrate your Workers to the ES Modules foramt:
```

File: content/workers/learning/migrate-to-module-workers.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+
+1.  Developer Platform products, such as [Durable Objects](/workers/learning/using-durable-objects/), and other Workers features require the module syntax.
```
Comment (irvinebroque) [inline content/workers/learning/migrate-to-module-workers.md]:
```suggestion
1.  Many products within Cloudflare's Developer Platform, such as [Durable Objects](/workers/learning/using-durable-objects/), and other features of Cloudflare Workers, require the ES Modules format.
```

File: content/workers/learning/migrate-to-module-workers.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+2.  Module Workers do not rely on any global bindings, which means the Workers runtime does not need to set up fresh execution contexts, making Module Workers safer and faster to run.
```
Comment (irvinebroque) [inline content/workers/learning/migrate-to-module-workers.md]: General note — I'm not sure we want to refer to these as "Module Workers", comes across more like this is a Capital Letter feature or product in and of itself — seems most specific to say "ES module format" — thoughts?

File: content/workers/learning/migrate-to-module-workers.md [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+
+Then becomes:
+
+```js
+async function triggerEvent(event) {
+  // Fetch some data
+  console.log('cron processed', event.scheduledTime);
+}
+
+const worker = {
```
Comment (irvinebroque) [inline content/workers/learning/migrate-to-module-workers.md]: This line could just be the default export? Simpler?

File: content/workers/learning/migrate-to-module-workers.md [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+To add support for module Workers to an existing Workers project, update your `wrangler.toml` file:
+
+```toml
+name = "my-worker"
+main = "src/index.js"
+
+# Uncomment if you have a build script.
+# [build]
+# command = "npm run build"
+```
```
Comment (irvinebroque) [inline content/workers/learning/migrate-to-module-workers.md:170]: Can remove this section

**Review by irvinebroque — COMMENTED** [review]
Some comments wondering about naming, see internal chat. In general LGTM! 🙌 thank you!

File: content/workers/learning/migrate-to-module-workers.md (hunk 7 above)
Comment (deadlypants1973) [reply content/workers/learning/migrate-to-module-workers.md]:
```suggestion
export default {
```
@irvinebroque does this work?

File: content/workers/learning/migrate-to-module-workers.md [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+  console.log('cron processed', event.scheduledTime);
+}
+
+export default {
+  async scheduled(event, env, ctx) {
+    ctx.waitUntil(triggerEvent(event));
+  },
+};
+
+export default worker;
```
Comment (WalshyDev) [inline content/workers/learning/migrate-to-module-workers.md]:
```suggestion
```

Comment (deadlypants1973) [inline content/workers/learning/migrate-to-module-workers.md]:
```suggestion
```

**Review by angelampcosta — APPROVED** [review]

**Pull request merged**